cmd/booster-bitswap/filters/blockfilter.go

```diff
@@ -0,0 +1,109 @@
+package filters
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"io"
+	"sync"
+
+	"github.com/ipfs/go-cid"
+	logging "github.com/ipfs/go-log/v2"
+	peer "github.com/libp2p/go-libp2p/core/peer"
+	"github.com/multiformats/go-multibase"
+)
+
+var log = logging.Logger("booster-bitswap")
+
+// BadBitsDenyList is the URL for well known bad bits list
+const BadBitsDenyList string = "https://badbits.dwebops.pub/denylist.json"
```

Comment:
are we able to support this denylist as a user-defined config, rather than a single centralized list?